api/handler.ts

```
@@ -0,0 +1,75 @@
+  return res.status(200).json({
+    message: 'Hello from root!',
+  });
+});
+
+app.get('/hello', (_req: Request, res: Response) => {
+  return res.status(200).json({
+    message: 'Hello from path!',
+  });
+});
+
+app.post('/register', async (req: Request<{}, {}, RegisterRequestBody>, res: Response) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)